<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T4selfgate_agata_psa_dlg</class>
 <widget class="QDialog" name="T4selfgate_agata_psa_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="_19">
     <item>
      <widget class="QLabel" name="textLabel1_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>&lt;font size=&quot;+2&quot;&gt;'Self Gate' name:&lt;/font&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>32767</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_extension">
       <property name="text">
        <string>extension</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>&lt;i&gt;(by changing this name - you CLONE of the old self gate)&lt;/i&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="spacer10">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="textLabel_invocation">
     <property name="text">
      <string>&lt;b&gt;The 'self gate' allows to use the agata psa data only when:&lt;/b&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="spacer12">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="_14">
     <item>
      <widget class="QCheckBox" name="checkBox_energy">
       <property name="text">
        <string>Energy of hit is in the following range:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="_15">
       <item>
        <widget class="QLineEdit" name="lineEdit_energy_low">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>32767</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_2_3_4">
         <property name="text">
          <string>-</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_energy_high">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>32767</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="_16">
     <item>
      <layout class="QHBoxLayout" name="_17">
       <item>
        <widget class="QCheckBox" name="checkBox_time">
         <property name="text">
          <string>Time value is in the following range:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_18">
         <item>
          <widget class="QLineEdit" name="lineEdit_time_low">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel1_2_3">
           <property name="text">
            <string>-</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_time_high">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer8_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>121</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <spacer name="spacer9">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>21</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Geometric position of a particular gamma hit</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="_3">
        <item>
         <widget class="QCheckBox" name="checkBox_x">
          <property name="text">
           <string>X position of the hit [mm]</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_4">
          <item>
           <widget class="QLineEdit" name="lineEdit_x_low">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>32767</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel1_2_3_2">
            <property name="text">
             <string>:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_x_high">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>32767</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="3">
       <spacer name="spacer18">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>150</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="_5">
        <item>
         <widget class="QCheckBox" name="checkBox_y">
          <property name="text">
           <string>Y position of the hit [mm]</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_6">
          <item>
           <widget class="QLineEdit" name="lineEdit_y_low">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>32767</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel1_2_3_2_2">
            <property name="text">
             <string>:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_y_high">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>32767</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="_7">
        <item>
         <widget class="QCheckBox" name="checkBox_z">
          <property name="text">
           <string>Z position of the hit [mm]</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_8">
          <item>
           <widget class="QLineEdit" name="lineEdit_z_low">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>32767</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel1_2_3_2_2_2">
            <property name="text">
             <string>:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_z_high">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>32767</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="1">
    <spacer name="spacer11">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>21</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>One agata detector consists of 36 segments, so &lt;br&gt;
in one event there may be many gamma &quot;hits&quot; registered.</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_mult">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Multiplicity of hits in this crystal  is in the range</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="_11">
       <item>
        <widget class="QLineEdit" name="lineEdit_mult_low">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>32767</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_2_3_4_3">
         <property name="text">
          <string>-</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_mult_high">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>32767</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer19">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>41</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="12" column="1">
    <spacer name="spacer11_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>21</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>textLabel2</zorder>
  <zorder>textLabel1</zorder>
  <zorder>textLabel_invocation</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>T4selfgate_agata_psa_dlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>T4selfgate_agata_psa_dlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
